### PR TITLE
Add buaa.edu.cn.xml

### LINKED ISSUE #17475: New Ruleset Request for buaa.edu.cn
[//]: # (Thank you for reporting an issue to HTTPS Everywhere.)
[//]: # (We welcome input from users on improving this project.)
[//]: #
[//]: # (Please help us by following this issue template.)
[//]: # (You can delete all blank lines and all lines starting)
[//]: # (with the comment marker, such as this one.)
[//]: #
[//]: # (Delete all but one of the following "Type" lines.)
[//]: # (Leave in the line that best describes the issue you are reporting.)
[//]: #
[//]: # (If you are submitting a new ruleset, please check the list at)
[//]: # (https://www.eff.org/https-everywhere/atlas/index.html and the open)
[//]: # (issues and pull requests to make sure it doesn't already exist.)

Type: new ruleset

[//]: # (If you are reporting a ruleset/website problem, include the top-level)
[//]: # (domain below. For example, if you want to report an issue about)
[//]: # ("one.example.com" and "two.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: example.com)
[//]: #
[//]: # (Be sure to remove the parenthesis and comment marker.  If you are only)
[//]: # (reporting an issue about "one.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: one.example.com)
[//]: #
[//]: # (Only include one top-level domain. If you have more than one top-level)
[//]: # (domain to report, such as both "example.com" and "example.org", open a)
[//]: # (new issue for each top-level domain.)

Domain: buaa.edu.cn

[//]: # (Include any other relevant information below. Thank you again for)
[//]: # (helping to improve HTTPS Everywhere.)

Beihang University, or Beijing University of Aeronautics and Astronautics (a. k. a. BUAA), founded in 25 Oct 1952, is one of the top universities in mainland China. In 25 Oct 2018, the day of the 66th anniversary of the university, the Network and Information Center of this university moved a great leap forward a more secure Internet by moving www.buaa.edu.cn, the university's Chinese main page, ev.buaa.edu.cn, the English main page, and news.buaa.edu.cn, the university's news center, to HTTPS. Alongside the websites above, a small number of the university's websites are also already enabled HTTPS, such as mail system (mail.buaa.edu.cn) , VPN system (e.buaa.edu.cn) , SSO system (sso.buaa.edu.cn and ucs.buaa.edu.cn), and Financial Department (cwc.buaa.edu.cn).

### PR DESCRIPTION
**Pull request by pipboy96**
Closes #17475.

Due to the nature of Chinese internet I want review from someone who is located in Mainland. @cschanaj I know you are located in HK, but can you run a VPS in Mainland? I don't know any VPS providers that work without ICP number.

### CODE REVIEW

**Comment by cschanaj** [comment]
@pipboy96 I don't have access to any VPS service in mainland.

**Comment by pipboy96** [comment]
@cschanaj Thanks for comment!

File: src/chrome/content/rules/buaa.edu.cn.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		- pay.icw
+
+	HTTP =/= HTTPS:
+		- vpn.nlsde
+
+	SSL handshake error:
+		- ucs
+
+	HTTP/2 error:
+		- www
```
Comment (J0WI) [inline src/chrome/content/rules/buaa.edu.cn.xml]: WFM

Comment (pipboy96) [reply src/chrome/content/rules/buaa.edu.cn.xml]: From Mainland IPs?

Comment (pipboy96) [reply src/chrome/content/rules/buaa.edu.cn.xml]: Seems to be fixed on their side.

**Pull request merged**